### DISCUSSION TITLE
Do i need a page.js file in the root folder of my app like app/page.js file ?

### DISCUSSION THREAD

**Discussion by harshagg21521** [vercel/next.js#69346] [App Router]
i do not have root route , my root route needs a dynamic value so i don't need  page.js file in my root folder . How can i achieve this ?

**Comment by Juneezee** [comment]
Only the Root Layout is required, see https://nextjs.org/docs/app/building-your-application/routing/layouts-and-templates#root-layout-required.

If your app doesn't need the `/` route, then you don't need to have a root `page.js`, i.e. `app/page.js`.

**Reply by harshagg21521** [reply]
Thanks for the reply , i need one more help ,actually  I've divided my app into three sections and I'm using route groups for this. I want to know if it's possible for a user to switch between these sections by simply changing the URL."

**Reply by Juneezee** [reply]
Route Groups are not included in the URL path, see https://nextjs.org/docs/app/building-your-application/routing/route-groups.

> I want to know if it's possible for a user to switch between these sections by simply changing the URL

What you want to achieve here is possible with the plain simple `/section1`, `/section2`, `/section3` routes. You don't need to use Route Groups unless you have special needs such as different root layouts for each group etc.

**Comment by harshagg21521** [comment]
I have three different layouts for three sections each section have different url like section1 have xavierchild.com (its the home page) , section2 have xavier.com/usertype/cityname (its the home page where usertype and cityname are dynamic), section3 have xavier.com/signin after signin it redirect to xavier.com/admin  , after deployment on ubuntu its only working for section1 which is xavierchild.com
